src/lib/button-area/_core.scss

```diff
@@ -0,0 +1,25 @@
+@use './token-utils' as *;
+
+@forward './token-utils';
+
+@mixin host {
+  display: block;
+  position: relative;
+}
+
+@mixin base {
+  position: relative;
+
+  outline: none;
+  user-select: none;
+  -webkit-appearance: none;
```

Comment:
No need to set `outline` and `appearance` here. The root element is just a `div` so the typical `button` overrides don't do anything. Also remove `user-select` — the button area is intended to wrap complex content that the user might want to select. There's code in the component to prevent a click if a selection was made.